TF-3410 Reduce color noise and user load

dab246 opened the pull request:
## Issue

#3410 

## Resolved




https://github.com/user-attachments/assets/3c83c4d4-71a8-43ee-b831-3f086119b882

hoangdat commented:
- @dab246  please rebase to `master`

dab246 commented:
> * @dab246  please rebase to `master`

Done

hoangdat commented on lib/features/base/mixin/message_dialog_action_mixin.dart:
```diff
@@ -185,24 +191,29 @@ mixin MessageDialogActionMixin {
+              fontWeight: FontWeight.w500,
+              color: AppColor.steelGray600,
+            ))
+            ..styleTextConfirmButton(actionStyle ?? const TextStyle(
+              fontSize: 17,
+              height: 24 / 17,
+              fontWeight: FontWeight.w500,
+              color: Colors.white,
+            ))
```
we should use as the typography define in figma to make it easy to change in the future.
```suggestion
            ..styleTextConfirmButton(actionStyle ?? Theme.of(context).textTheme.bodyLarge))
```

<img width="519" alt="image" src="https://github.com/user-attachments/assets/a40aecdb-573b-4fe1-949c-4bf12179ee59" />

And define details of bodyLarge in TextTheme(), then use it with ThemeData

hoangdat replied:
```
static var fallbackTextTheme = const TextTheme(
    bodyLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: -0.15,
    ),
    bodyMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.25,
    ),
    bodySmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.4,
    ),
    labelLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.1,
    ),
    labelSmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.5,
    ),
    displayLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w700,
    ),
    displayMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: 0.4,
    ),
    displaySmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: 0.4,
    ),
    headlineMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
    ),
    headlineSmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: 0.4,
    ),
    headlineLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      fontSize: 32,
    ),
    titleLarge: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w600,
      letterSpacing: -0.15,
    ),
    titleMedium: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.15,
    ),
    titleSmall: TextStyle(
      fontFamily: 'Inter',
      fontWeight: FontWeight.w500,
      letterSpacing: 0.1,
    ),
  );
```

dab246 replied:
Updated

hoangdat commented:
- [ ] list email item in ThreadView was not applied correct font 
![image](https://github.com/user-attachments/assets/c878e943-d5ec-48b2-935f-b10b56768968)

<img width="1356" alt="Screenshot 2025-03-12 at 10 42 25" src="https://github.com/user-attachments/assets/8bc7a7fc-cf27-4ec3-893f-31e7850a3cf1" />

hoangdat commented:
- [ ] wrong font size, it must be `Body/Body2`
![image](https://github.com/user-attachments/assets/d88a51f8-62ad-476e-bb91-85ac86b044a8)

hoangdat commented:
- [ ] should be `M3/body/medium2` size 14, weight 600

![image](https://github.com/user-attachments/assets/60c61525-4481-415d-a09d-71fbef72af30)

hoangdat commented:
- [ ] color of next/previous button should be gray...
![Screenshot 2025-03-12 at 10 58 03](https://github.com/user-attachments/assets/74285d2d-1636-4d37-b9f7-124d91a18a26)

hoangdat commented:
- [ ] should be 50x50
![image](https://github.com/user-attachments/assets/082886f0-33a4-4b1e-86f9-5cd3b24f8f55)

hoangdat commented:
- [ ] should be Heading/Heading small

![image](https://github.com/user-attachments/assets/8d2d7675-b695-4b31-9e65-ccd40d2e6bf4)

hoangdat commented:
- [ ] should be `Body/ body 1`

![image](https://github.com/user-attachments/assets/5aa8aa31-61ce-4086-992a-96e200ef9747)

hoangdat commented:
- [ ] should use `Heading/Heading small`

![image](https://github.com/user-attachments/assets/6a8956aa-23ed-4f9f-b25a-f27a2cd9ebab)

hoangdat commented:
- [ ] size 17, weight 500

![image](https://github.com/user-attachments/assets/3213f382-9117-4e8f-9cf9-d6aeeeb03238)

dab246 commented:
> * [x]  list email item in ThreadView was not applied correct font
>
> * [x]  wrong font size, it must be `Body/Body2`
>
> * [x]  should be `M3/body/medium2` size 14, weight 600
> 
> * [x]  color of next/previous button should be gray...
> 

Fixed

> * [x]  should be 50x50
>

What you see is `Align`. Its size is exactly 50px

<img width="1438" alt="Screenshot 2025-03-12 at 14 34 02" src="https://github.com/user-attachments/assets/7bdf2ecb-9693-4f8f-bca8-5bdc1cc2f481" />




> * [x]  should be Heading/Heading small
> * [x]  apply new design for these components too
> 
> * [x]  should be `Body/ body 1`
> 
> * [x]  should use `Heading/Heading small`
> 
> * [x]  size 17, weight 500
> 


Fixed.
